Yasgui/packages/yasr/src/plugins/extended-table/extended-table.ts

```diff
@@ -0,0 +1,28 @@
+import Table from "../table";
+import Parser from "../../parsers";
+import Yasr from "@triply/yasr";
+
+export class ExtendedTable extends Table {
+  public label = "Extended_Table";
+  public priority = 11;
+
+  private readonly getCellContentCustom?: (
+    binding: Parser.BindingValue,
+    prefixes?: { [label: string]: string }
+  ) => string;
+
+  constructor(yasr: Yasr) {
+    super(yasr);
+    if (yasr.config.externalPluginsConfigurations) {
+      const pluginConfiguration = yasr.config.externalPluginsConfigurations.get("extended_table");
```

Comment:
We decided to extend original table plugin instead to change source code.
This is ontotext implementation of table plugin, but we decided to not use ontotext in names.
For now, we override "getCellContent" function only, but this can be change in the future.